src/kiota/KiotaHost.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using Kiota.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Kiota {
+    public static class KiotaHost {
+        public static RootCommand GetRootCommand()
+        {
+            var configuration = LoadDefaultConfiguration();
+            var outputOption = new Option("--output", "The ouput path of the folder the code will be generated in.") { Argument = new Argument<string>(() => "./output") };
+            outputOption.AddAlias("-o");
+            
+            var languageOption = new Option("--language", "The language to generate the code in.") { Argument = new Argument<GenerationLanguage?>(() => GenerationLanguage.CSharp) };
+            languageOption.AddAlias("-l");
+            AddEnumValidator<GenerationLanguage>(languageOption.Argument, "language");
+            var classOption = new Option("--class-name", "The class name to use the for main entry point") { Argument = new Argument<string>(() => "GraphClient") };
```

Comment:
`The class name to use the for main entry point` becomes
`The class name to use for the core client class. The default core client class name is GraphClient.`

Reply:
Do we want to have a default class name as we will want to make this an API agnostic tool? I think we make this a required option without a default.

Reply:
I wanted to provide consistency with all other arguments (everything has a default). And the command line SDK will list out the default value already when displaying help.
I'm open to changing it to "APIClient" or something more generic, thoughts?